sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuAvroScan.scala

```diff
@@ -777,6 +777,7 @@ class GpuMultiFileCloudAvroPartitionReader(
 
           val bufAndSize: Array[(HostMemoryBuffer, Long)] = if (readDataSchema.isEmpty) {
```

Comment:
if readDataSchema is empty, can we return the value earlier without reading out the avro data?

Reply:
Tracked by https://github.com/NVIDIA/spark-rapids/issues/6219